scripts/workers/submission_worker.py

```diff
@@ -161,6 +160,24 @@ def delete_zip_file(download_location):
         traceback.print_exc()
 
 
+def delete_submission_data_directory(location):
+    """
+        Helper function to delete submission data from location `location`
+
+        Arguments:
+            location {[string]} -- Location of directory to be removed.
+    """
+    try:
+        shutil.rmtree(location)
+    except Exception as e:
+        logger.error(
```

Comment:
Instead of logger.error, you can directly use logger.exception and that will automatically print the message that you pass in followed by the traceback. Then you can also get rid of the `traceback.print_exc()`